src/clients/detector/text_chat.rs

```diff
@@ -109,6 +109,7 @@ pub struct ChatDetectionRequest {
     /// Chat messages to run detection on
     pub messages: Vec<Message>,
 
+    /// Detector parameters
```

Comment:
```suggestion
    /// Detector parameters (available parameters depend on the detector)
```
In case this makes sense, could you also update it in other classes?